docs/kubernetes.adoc

```diff
@@ -240,6 +240,31 @@ a possible location of resources needed to be accessed during tests, usually wit
 you are testing made some changes in state of services in another namespaces, you can specify the secondary
 namespace in a field of @Named annotation to reach them.
 
+[NOTE]
+====
+Necessary configuration to avoid permission conflicts for creating namespaces.
+
+1. If using `MiniShift` to spin up a local Kubernetes cluster, login in as `admin` to create the default namespace or
+use `current` or `existing` namespace configured using properties `namespace.use.current` and `namespace.use.existing`
+respectively if logged in as any other user.
+
+2. In case of remote cluster use `current` or `existing` namespace for the user authenticated by token using property
+`cube.auth.token` or by username and password using properties `cube.username` and `cube.password` respectively.
+
```

Comment:
Can we add about user can overwrite this property by setting system  property while running tests? So they can avoid putting `cube.username`, `cube.password`, `cube.auth.token` in `arquillian.xml`